pandas/_libs/tslibs/util.pxd

```diff
@@ -37,7 +37,6 @@ from numpy cimport int64_t
 
 cdef extern from "numpy/arrayobject.h":
     PyTypeObject PyFloatingArrType_Type
-    int _import_array() except -1
```

Comment:
Was this just dead code?

Reply:
yes.  I think i introduced it a while back as part of a failed attempt to get rid of the NPY_DEPRECATED_API warnings.